### ISSUE TITLE
Framerate independence

### ISSUE THREAD

**Issue by ABoschman** [amethyst/space-menace#43]
At the moment, whenever there is a drop in frames the characters noticeably slow down. When running in debug mode, it is common (at least on my laptop) for framerates to occasionally drop down to 30-40. This causes a very noticeable and irritating jank in the characters' movements.

It would be desirable if the movements and animations advanced as a function of time passed, rather than number of frames passed.

- Entity velocities should be expressed in m/s, rather than m/frame
- Translation should be updated by adding velocity times delta seconds

I'd like to give this a go, if nobody is working on this yet. Any input is welcome. :)

**Comment by krankur** [comment]
@ABoschman Thanks for logging this issue. This is a must have feature. Though somehow I neither logged an issue for it nor worked on it (kept putting it off actually). Glad that you brought it up and want to work on it. Much appreciated. Assigned it to you.

On a different note, I noticed that your name doesn't show in the contributor's list. It seems like you have set a different email id for this repository than the one you have set for GitHub account. You might want to use same email id at both places, in case you want to be listed in the contributors list. In case its intentional, please ignore. Just want to make sure that you are aware of it.

**Comment by Jazarro** [comment]
No, I didn't know it worked like that. I was wondering why I didn't show up in the contributors list. GitHub does seem to identify those commits as mine. Weird.

**Comment by krankur** [comment]
Yeah, even I had faced the same issue. But seems like you have updated your email id as the contributors count has gone up to 6 and it shows your name if you hover over the count. GitHub still doesn't show your name when on the contributors page though, which is strange. But it should show up there as well if you make any new commits now onward. At least that's what I observed in my case.
